docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,205 @@
+---
+id: adjusting-bpmn-models
+title: Adjusting BPMN models
+description: "Migrate process solutions developed for Camunda Platform 7 to run them on Camunda Platform 8."
+keywords:
+  [
+    Camunda 8,
+    Camunda 7,
+    migration guide,
+    transition,
+    transition guide,
+    Camunda Platform 7,
+  ]
+---
+
+To migrate BPMN process models from Camunda Platform 7 to Camunda Platform 8, you must adjust them:
+
+- The namespace of extensions has changed (from `http://camunda.org/schema/1.0/bpmn` to `http://camunda.org/schema/zeebe/1.0`)
+- Different configuration attributes are used
+- Camunda Platform 8 has a _different coverage_ of BPMN elements (see [Camunda Platform 8 BPMN coverage](/components/modeler/bpmn/bpmn-coverage.md) vs [Camunda Platform 7 BPMN coverage](https://docs.camunda.org/manual/latest/reference/bpmn20/)), which might require some model changes. Note that the coverage of Camunda Platform 8 will increase over time.
+
+The following sections describe what the existing [Camunda Platform 7 to Camunda Platform 8 migration tooling](https://github.com/camunda-community-hub/camunda-7-to-8-migration/) does by BPMN symbol and explain unsupported attributes.
```

Comment:
Let's go a step deeper and link to the backend-diagram-converter inside the migration repo here.